src/EditorFeatures/Core/Diagnostics/AbstractPushOrPullDiagnosticsTaggerProvider.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.CodeAnalysis.Diagnostics;
 internal static class DiagnosticTaggingOptions
 {
     public static readonly Option2<bool> PullDiagnosticTagging = new(
-        "DiagnosticTaggingOptions_PullDiagnosticTagging", defaultValue: true);
+        "dotnet_tagging_pull_diagnostic", defaultValue: true);
```

Comment:
>dotnet_tagging_pull_diagnostic [](http://example.com/codeflow?start=9&length=30)

`dotnet_pull_diagnostic_tagging` would read better, though it doesn't matter that much since this is going to go away at some point.